pages/blog/swift-graphql.mdx

```diff
@@ -0,0 +1,160 @@
+import withArticle from '../../ui/blog/article';
+
+<!-- Configuration -->
+
+export default withArticle({
+  title: 'SwiftGraphQL - A GraphQL client that lets you forget about GraphQL',
+  author: 'maticzav',
+  tags: ['graphql', 'swift', 'client', 'swift-graphql'],
+  date: '2021-03-25',
+  description: 'An introduction article for SwiftGraphQL client.',
```

Comment:
maybe something sexier?
LIke "A new GraphQL Client for Swift Lovers" or something....